log4j-core-test/src/test/java/org/apache/logging/log4j/core/appender/SyslogAppenderTestBase.java

```diff
@@ -43,13 +43,16 @@
 import org.apache.logging.log4j.util.Strings;
 import org.junit.jupiter.api.BeforeAll;
 
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.awaitility.Awaitility.await;
+
 @UsingStatusListener
 abstract class SyslogAppenderTestBase {
     protected static final String line1 =
             "TestApp - Audit [Transfer@18060 Amount=\"200.00\" FromAccount=\"123457\" ToAccount=\"123456\"]"
                     + "[RequestContext@18060 ipAddress=\"192.168.0.120\" loginId=\"JohnDoe\"] Transfer Complete";
     protected LoggerContext ctx = LoggerContext.getContext();
-    protected static final int DEFAULT_TIMEOUT_IN_MS = 100;
+    protected static final int DEFAULT_TIMEOUT_IN_MS = 500;
```

Comment:
We can be a little bit generous here, Awaitility's default poll period is 100ms anyway.

```suggestion
    protected static final int DEFAULT_TIMEOUT_IN_MS = 3000;
```